generated/nixnet/nixnet_service.h

```diff
@@ -31,6 +31,8 @@ struct NiXnetFeatureToggles
   bool is_enabled;
 };
 
+enum TypeIdentifier { db_ref_array_, string_, boolean_, u64_, u32_, db_ref_, f64_, u8_array_, u32_array_, parent_ref_, structure_, i32_, string_array_, dev_ref_array_, intf_ref_array_, dev_ref_,  };
```

Comment:
I don't see any references to the types {parent_ref_, structure_, dev_ref_array_, intf_ref_array_, dev_ref_} in the custom service implementations. I was curious if we have follow up work to handle these property types.

Reply:
There will be a follow up PR for adressing all these typeidentifiers that you have mentioned. This PR handlee support for most commonly used types. Those will be added in next PR.

Reply:
Apart from structures, the others will most probably be changed to db_ref or db_ref_ in enums.py and hence won't be autogenerated here as well. If not then they will be addressed in next PR else I'll make that change in the same PR. I have put out an email to the XNET expert to clarify if these can be used as db_ref or db_ref_array or not.